processor/batchprocessor/README.md

```diff
@@ -29,6 +29,9 @@ regardless of size.
   `0` means no upper limit of the batch size.
   This property ensures that larger batches are split into smaller units.
   It must be greater than or equal to `send_batch_size`.
+- `metadata_keys` (default = empty): When set, this processor will
+  create one batcher instance per distinct combination of values in
+  the `client.Metadata`.  See notes about metadata batching below.
```

Comment:
Can you add `metadata_cardinality_limit` to this list of options?

Reply:
Done.